modules/restarting-sensor-in-the-secured-cluster.adoc

```diff
@@ -6,7 +6,7 @@
 [id="restarting-sensor-in-the-secured-cluster_{context}"]
 = Restarting Sensor in the secured cluster
 
-If you installed the Compliance Operator before installing {product-title-short}, you need to restart Sensor in the secured cluster either by using the command-line interface (CLI) or the user interface (UI).
+If you installed the Compliance Operator after installing {product-title-short}, you need to restart Sensor in the secured cluster either by using the command-line interface (CLI) or the user interface (UI).
```

Comment:
I don't think this step will be necessary in newer versions of ACS, but I believe it will be for 4.4.

@dashrews78 do you know if CO detection will be released in a patch update to 4.4?

Reply:
No that will not be in a 4.4 patch.  Only 4.5 forward.